EveripediaNetwork/ep-ui
# Implemented distinct styling for active links

# Add distinct style for active nav items

_Implements a distinct style for active navigation items in the Navbar component._

## How should this be tested?

1. Run the application.
2. Navigate through different pages.
3. Check that the active navigation items have a distinct style.

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3037
